recipes/yaehmop/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "yaehmop" %}
+{% set version = "2018.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/greglandrum/yaehmop.git
+  sha256: 8517627fa7ca14af71064bbff32f1d6077459aa0759923cc3486234b7996b218
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+
+test:
+  commands:
+    - eht_bind -v
+
+about:
+  home: https://github.com/greglandrum/yaehmop
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: license.txt
+  summary: 'Yet Another extended Hueckel Molecular Orbital Package'
+  doc_url: https://github.com/greglandrum/yaehmop/blob/master/docs/bind_manual.pdf
+  dev_url: https://github.com/greglandrum/yaehmop
+  
+extra:
+  recipe-maintainers:
+    - richardjgowers
+    - micaela-matta
+    - greglandrum
```

Comment:
We need to make sure @greglandrum is ok being a maintainer? They should chime in here to confirm or deny!

Reply:
@beckermr got the thumbs up on gh